aten/src/ATen/Context.h

```diff
@@ -114,12 +114,42 @@ class CAFFE2_API Context {
   void setDeterministicCuDNN(bool);
   bool deterministic() const;
   void setDeterministic(bool);
+
+  // Throws an error if `Context::deterministic()` is true. This method should be called by
+  // operations that act nondeterministically. Most of the time, this method should be called
+  // like this:
+  //  at::globalContext().alertNotDeterministic("<name of operation>");
+  //
+  // All operations that call this method must be documented in torch/__init__.py:set_deterministic()
+  //
+  // All callsites should have a comment explaining why the operation is nondeterministic.
+  //
+  // To help enforce full documentation, all callsites should also have the following note:
+  // NOTE: If you copy and paste this alert to a different operation, you must add a
```

Comment:
Would it be OK if this note was just a one-line? 

"See the note [Writing Nondeterministic Operations] when implementing a nondeterministic operation"?

That might make maintenance easier in the future.